_posts/2019-03-01-preparing-the-bagwell-award-2019.md

```diff
@@ -0,0 +1,31 @@
+---
+category: blog
+title: "Preparing the 2019 Phil Bagwell Community Award"
```

Comment:
please change the title to: "Phil Bagwell Nominations"

Reply:
Done.